WordPress/Classes/ViewRelated/Post/PostPreviewViewController.m

```diff
@@ -154,7 +154,6 @@ - (void)refreshWebView
 - (void)webViewDidFinishLoad:(UIWebView *)awebView
 {
     DDLogMethod();
-    [self stopLoading];
```

Comment:
@diegoreymendez This works great for me even on slow connections. However, I think we should still call the `[self stopLoading]` but **not call** the `self.webView stopLoading` part of it:

https://github.com/wordpress-mobile/WordPress-iOS/blob/cc53bb295e3c5759715028bc9666814b42a98fe6/WordPress/Classes/ViewRelated/Post/PostPreviewViewController.m#L123-L128

The other lines of `[self stopLoading]` are for resetting the header. Right now, the header is stuck at _Loading_. This has been going on for a few minutes after everything was loaded: 

<img src="https://user-images.githubusercontent.com/198826/63182226-53886180-c00f-11e9-857d-15957f64fd68.png" width="320">

Reply:
Fixed.